docs/transpile/ai-transpiler-passes.mdx

```diff
@@ -100,4 +100,9 @@ The following custom collection passes for Cliffords, Linear Functions and Permu
 - *CollectLinearFunctions*: Collects blocks of `SWAP` and `CX` as `LinearFunction` objects and stores the original sub-circuit to compare against it after synthesis.
 - *CollectPermutations*: Collects blocks of `SWAP` circuits as `Permutations`.
 
-These custom collection passes limit the sizes of the collected sub-circuits so that they are supported by the AI synthesis passes, so it is recommended to use them after the routing passes and before the synthesis passes for a better overall optimization.
+These custom collection passes limit the sizes of the collected sub-circuits so they are supported by the AI synthesis passes. Therefore, it is recommended to use them after the routing passes and before the synthesis passes for a better overall optimization.
+
+
+## Citation
+
+If you use any AI feature from the Qiskit transpiler service in your research, don't forget to use [the recommended citation](./qiskit-transpiler-service#citation)!
```

Comment:
This was a bad suggestion - we want to use a consistent professional tone in our docs

```suggestion
If you use any AI feature from the Qiskit transpiler service in your research, use [the recommended citation](./qiskit-transpiler-service#citation).
```